test/integration_test.exs

```diff
@@ -216,26 +642,21 @@ defmodule IntegrationTest do
     assert response.status == 200
   end
 
-  test "creates table inside AWS Athena's database with workgroup" do
-    opts = [
-      access_key_id: System.fetch_env!("AWS_ACCESS_KEY_ID"),
-      secret_access_key: System.fetch_env!("AWS_SECRET_ACCESS_KEY"),
-      region: System.fetch_env!("AWS_REGION"),
-      database: "default",
-      workgroup: "primary"
-    ]
-
-    req = Req.new(http_errors: :raise) |> ReqAthena.attach(opts)
-    response = Req.post!(req, athena: @create_table)
-    result = response.body
+  # TODO: check why it's not working only with "workgroup"
```

Comment:
Is this needed to ship a new version or we can address it later?

Reply:
Since this is happening on main, I will ignore it for now.